config/environments/development.rb

```diff
@@ -63,4 +63,20 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  # SMTP configuration
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    user_name: Rails.application.credentials.mailtrap[:username],
+    password: Rails.application.credentials.mailtrap[:password],
+    address: 'smtp.mailtrap.io',
```

Comment:
Like in the first comment, all this things should be in the secrets file.
Check this link: http://guides.rubyonrails.org/upgrading_ruby_on_rails.html#config-secrets-yml

Reply:
You can use the credentials here